packages/rest-explorer/src/rest-explorer.controller.ts

```diff
@@ -41,18 +39,24 @@ export class ExplorerController {
     explorerConfig: RestExplorerConfig = {},
     @inject(RestBindings.BASE_PATH) private serverBasePath: string,
     @inject(RestBindings.SERVER) private restServer: RestServer,
-    @inject(RestBindings.Http.REQUEST) private request: Request,
     @inject(RestBindings.Http.CONTEXT) private requestContext: RequestContext,
-    @inject(RestBindings.Http.RESPONSE) private response: Response,
   ) {
     this.relativeOpenApiSpecEndpoint =
       explorerConfig.relativeOpenApiSpecEndpoint !== false;
     this.openApiSpecUrl = this.getOpenApiSpecUrl(restConfig);
   }
 
   indexRedirect() {
-    const url = this.request.originalUrl || this.request.url;
-    this.response.redirect(301, url + '/');
+    let url =
+      this.requestContext.request.originalUrl ||
```

Comment:
Repeating `this.requestContext.` prefix is tedious, have you considered the following refactoring?

```ts
const {request, response} = requestContext;

let url = request.originalUrl || request.url;
// etc.
```